### PR TITLE
Cleanup tempfiles

### PR DESCRIPTION
**Pull request by zhijianli88**

### CODE REVIEW

File: pkg/logging/file_test.go [hunk 1]
```diff
@@ -50,9 +53,12 @@ func TestJSONFileLogger_basic(t *testing.T) {
 }
 
 func TestJSONFileLogger_parallel(t *testing.T) {
-	f, err := ioutil.TempFile("", "")
+	f, err := ioutil.TempFile("", "logging-")
 	testutil.Ok(t, err)
-	defer f.Close()
+	defer func() {
+		f.Close()
```
Comment (codesome) [inline pkg/logging/file_test.go:59]: Let us also use `testutil.Ok` for this

File: pkg/logging/file_test.go [hunk 2]
```diff
@@ -24,9 +24,12 @@ import (
 )
 
 func TestJSONFileLogger_basic(t *testing.T) {
-	f, err := ioutil.TempFile("", "")
+	f, err := ioutil.TempFile("", "logging-")
 	testutil.Ok(t, err)
-	defer f.Close()
+	defer func() {
+		f.Close()
```
Comment (codesome) [inline pkg/logging/file_test.go:30]: Let us also use `testutil.Ok` for this

File: pkg/logging/file_test.go [hunk 3]
```diff
@@ -24,9 +24,12 @@ import (
 )
 
 func TestJSONFileLogger_basic(t *testing.T) {
-	f, err := ioutil.TempFile("", "")
+	f, err := ioutil.TempFile("", "logging-")
```
Comment (codesome) [inline pkg/logging/file_test.go:27]:
```suggestion
	f, err := ioutil.TempFile("", "logging")
```

File: pkg/logging/file_test.go [hunk 4]
```diff
@@ -50,9 +53,12 @@ func TestJSONFileLogger_basic(t *testing.T) {
 }
 
 func TestJSONFileLogger_parallel(t *testing.T) {
-	f, err := ioutil.TempFile("", "")
+	f, err := ioutil.TempFile("", "logging-")
```
Comment (codesome) [inline pkg/logging/file_test.go:56]:
```suggestion
	f, err := ioutil.TempFile("", "logging")
```

**Comment by zhijianli88** [comment]
hi @codesome any further comments for this PR?

**Review by codesome — APPROVED** [review]

**Pull request merged**